airbyte-webapp/src/components/ui/PillSelect/PillButton.tsx

```diff
@@ -47,9 +49,15 @@ export const PillButton: React.FC<PillButtonProps> = ({
       {Children.map(arrayChildren, (child, index) => (
         <>
           <div key={index} className={styles.labelContainer}>
-            <Text as="span" size="xs" className={styles.text}>
-              {child}
-            </Text>
+            {withOverflowTooltip ? (
+              <TextWithOverflowTooltip as="span" size="xs" className={styles.text}>
+                {child}
+              </TextWithOverflowTooltip>
+            ) : (
+              <Text as="span" size="xs" className={styles.text}>
+                {child}
+              </Text>
+            )}
```

Comment:
I think this is may be the wrong level for the tooltip.
My PK selection cell won't show overflow unless I select 2 or more items.

Reply:
D'oh!  It's the `as="span"`.  Can't overflow text in a span :)